### PR TITLE
fix(nx): Ignore the debugger attached message in node

### LINKED ISSUE #11130: Fails to serve swc executor in VS Code.
## Current Behavior
I have created a default Launch config in VS Code to run my app which is using the @nrwl/js:swc executor.
During execution, VS Code automatically attach a debugger, and SWC will print out `Debugger attached.` to stdout. See [this](https://github.com/nodejs/node/issues/34799#issuecomment-674782369) comment.

In the `compile-swc` file, `stderrOnData` will be called with the string `Debugger attached.`, and will mark the compilation as failed.

## Expected Behavior
The swc executor should ignore the string `Debugger attached.`, and should consider the compilation successful.

## Steps to Reproduce
Add a project with the executor `"executor": "@nrwl/js:swc",`.
Launch it with a debugger, this is easy in VS Code.

This issue may not be prioritized if details are not provided to help us reproduce the issue.

### Failure Logs
```
Debugger attached. // This is my own console.log
There was an error with the build. See above.
```

### Environment
```
 >  NX   Report complete - copy this into the issue template

   Node : 16.15.0
   OS   : darwin arm64
   yarn : 3.2.0
   
   nx : 14.4.2
   @nrwl/angular : Not Found
   @nrwl/cypress : 14.4.2
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.4.2
   @nrwl/eslint-plugin-nx : 14.4.2
   @nrwl/express : Not Found
   @nrwl/jest : 14.4.2
   @nrwl/js : 14.4.2
   @nrwl/linter : 14.4.2
   @nrwl/nest : Not Found
   @nrwl/next : 14.4.2
   @nrwl/node : 14.4.2
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 14.4.2
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.4.2
   @nrwl/web : 14.4.2
   @nrwl/workspace : 14.4.2
   typescript : 4.7.4
   ---------------------------------------
   Community plugins:
         @nxext/svelte: 14.0.0
         @nxext/vite: 14.0.1
```

### PR DESCRIPTION
**Pull request by jean343**
## Current Behavior
Node will print out `Debugger attached.` to stderr when a debugger is attached, thus marking the build as failed.

## Expected Behavior
The message should be ignored.

## Related Issue(s)

Fixes #11130

### CODE REVIEW

File: packages/js/src/utils/swc/compile-swc.ts [hunk 1]
```diff
@@ -148,6 +148,9 @@ export async function* compileSwcWatch(
       };
 
       stderrOnData = (err?: any) => {
+        if(err.includes("Debugger attached.")) {
+          return;
+        }
         process.stderr.write(err);
```
Comment (AgentEnder) [inline packages/js/src/utils/swc/compile-swc.ts:154]: We should still log the Debugger attached message, so it should probably be reordered:

```suggestion
        process.stderr.write(err);
        if(err.includes("Debugger attached.")) {
          return;
        }
```

**Review by AgentEnder — CHANGES_REQUESTED** [review]
Looks good, just a minor change:

**Comment by jean343** [comment]
> Looks good, just a minor change:

Love it, thank you!

**Comment by nartc** [comment]
@jean343 can you rebase and push real quick?

**Comment by jean343** [comment]
@nartc I have rebased the branch and formatted the source code appropriately.
cheers

**Comment by nartc** [comment]
@jean343 thank you. I'll check the failed CI after it finishes running.

**Pull request merged**